tests/unittests/onnxImporterTest.cpp

```diff
@@ -235,3 +235,36 @@ TEST(onnx, importDotProduct) {
   auto *batchedReduceAdd = llvm::cast<BatchedReduceAddNode>(saveInput);
   ASSERT_TRUE(llvm::isa<MulNode>(batchedReduceAdd->getBatch()));
 }
+
+/// Test loading Sum with one input and one output
+TEST(onnx, importSum1) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  std::string netFilename("tests/models/onnxModels/sum1.onnxtxt");
+
+  Context ctx;
+  Placeholder *output;
+  {
+    Tensor x(ElemKind::FloatTy, {3});
+    x.getHandle() = {1, 2, 3};
+    ONNXModelLoader onnxLD(netFilename, {"x"}, {&x.getType()}, *F);
+    output = onnxLD.getSingleOutput();
+
+    ctx.allocate(mod.getPlaceholders());
+    updateInputPlaceholdersByName(ctx, &mod, {"x"}, {&x});
+  }
+
```

Comment:
It'd be nice to have some checks of the graph structure here since it's an interesting net.  Let me suggest something like
```
// Verify structure: input -> Save -> output
ASSERT_EQ(mod.getPlaceholders().size(), 2);
ASSERT_EQ(F->getNodes().size(), 1);
auto *save = getSaveNodeFromDest(output);
ASSERT_TRUE(llvm::isa<Placeholder>(save->getInput().getNode()));
```